api/server/router/image/image_routes.go

```diff
@@ -241,6 +241,7 @@ func (s *imageRouter) postImagesLoad(ctx context.Context, w http.ResponseWriter,
 	output := ioutils.NewWriteFlusher(w)
 	defer output.Close()
 	if err := s.backend.LoadImage(r.Body, output, quiet); err != nil {
+		output.WriteHeader(http.StatusUnprocessableEntity)
```

Comment:
How are we coming to the conclusion that this is the correct status?
Also I think everywhere else we just use 400 for user errors.